src/customReplacer.js

```diff
@@ -12,7 +16,7 @@ module.exports = (template, replaceData) => {
       template = template
         .split(replaceData.search)
         .join(
-          replaceData.eval ? eval(replaceData.replace) : replaceData.replace
+          replaceData?.eval ? eval(replaceData.replace) : replaceData.replace
```

Comment:
```suggestion
          replaceData.eval ? eval(replaceData.replace) : replaceData.replace
```
`replaceData` variable already exists. `?` will not serve purpose here.

Reply:
Bruh, why don't you request all these changes at a time?

Reply:
I asked you to make the code as it was. You still kept that `?`. Wasn't expecting that!